Program Changes Cag642

cag642 opened the pull request:
Program changes given the comments from DJ.

bjones1 reviewed (changes requested):
This pull request should be sent to the `main` branch, not the `dh24540` branch. Please review and resubmit.

cag642 commented:
I have changed the merge to bjones1 : main.

cag642 commented:
I have revised the files. I removed the comments left about formatting and fixed those. There are two comments I left, so that you can see what was recommended and why I did not change them. 

I am requesting a new review.

bjones1 commented on thorgame/dh2540/The Power of Thor.cpp:
```diff
@@ -1,28 +1,37 @@
-// # *The Power of Thor*
```
Leave this file unchanged, since it's not your code.

bjones1 commented on PowerOfThor/cag642/Thor (1).cpp:
```diff
@@ -1,6 +1,11 @@
 // # Thor Program
 
```
Rename this to `Thor.cpp` instead of `Thor (1).cpp`.

bjones1 commented:
There are still edits to other's files -- chat with me to get this fixed.

cag642 commented:
I am so sorry, I just realized I never fixed this. I went into my fork and changed the DH file to what he has in main of the normal repo. For some reason that commit is not adding to this pull request though.

bjones1 commented:
I'd be happy to help -- when are you available?

cag642 commented:
I am available any time before 2:45

From: Bryan A. Jones ***@***.***>
Sent: Thursday, October 24, 2024 11:00 AM
To: bjones1/literate-programming-github-fall-2024 ***@***.***>
Cc: Gernaat, Cole ***@***.***>; Author ***@***.***>
Subject: Re: [bjones1/literate-programming-github-fall-2024] Program Changes Cag642 (PR #24)


I'd be happy to help -- when are you available?

—
Reply to this email directly, view it on GitHub<https://github.com/bjones1/literate-programming-github-fall-2024/pull/24#issuecomment-2435680717>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BLGBUSR3I67PP7XER7A2I6TZ5EKQFAVCNFSM6AAAAABPQQ5AHSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIMZVGY4DANZRG4>.
You are receiving this because you authored the thread.Message ID: ***@***.******@***.***>>

cag642 commented:
I am leaving for lunch, I will be back in 30 minutes.

cag642 commented:
Restarting with a new pull request
